### PR TITLE
Read sgs

### PR DESCRIPTION
**Pull request by YairSlobodin1**
1st bullet of #191

### CODE REVIEW

File: pkg/io/confio/parse_sgs.go [hunk 1]
```diff
@@ -5,8 +5,182 @@ SPDX-License-Identifier: Apache-2.0
...
+// temporary - first version of optimization requires local = 0.0.0.0/32
+func verifyLocalValue(ipAddrs *netset.IPBlock) (*netset.IPBlock, error) {
+	if !ipAddrs.Equal(netset.GetCidrAll()) {
+		return nil, fmt.Errorf("only 0.0.0.0/32 CIDR block is supported for local values")
+	}
+	return ipAddrs, nil
+}
+
+func translateProtocolTCPUDP(rule *vpcv1.SecurityGroupRuleSecurityGroupRuleProtocolTcpudp) (netp.Protocol, error) {
+	isTCP := *rule.Protocol == "tcp"
```
Comment (zivnevo) [inline pkg/io/confio/parse_sgs.go:182]:
```suggestion
	isTCP := *rule.Protocol == vpcv1.SecurityGroupRuleSecurityGroupRuleProtocolTcpudpProtocolTCPConst
```

**Review by zivnevo — CHANGES_REQUESTED** [review]
Well structured code!
Just a few tiny things

File: pkg/io/confio/parse_sgs.go [hunk 2]
```diff
@@ -5,8 +5,182 @@ SPDX-License-Identifier: Apache-2.0
...
+	}
+
+	result := ir.NewSGCollection()
+	for _, sg := range config.SecurityGroupList {
+		inbound, outbound, err := translateSGRules(&sg.SecurityGroup)
+		if err != nil {
+			return nil, err
+		}
+		if sg.Name == nil || sg.VPC == nil || sg.VPC.Name == nil {
+			continue
```
[the remark this answers is not in the document]
Comment (YairSlobodin1) [reply pkg/io/confio/parse_sgs.go:34]: Im not sure what should I write in the warning

[the remark this answers is not in the document]
Comment (zivnevo) [reply pkg/io/confio/parse_sgs.go:34]: missing SG/VPC name

**Review by zivnevo — APPROVED** [review]

**Review by zivnevo — APPROVED** [review]

**Pull request merged**